docs/UserGuide.md

```diff
@@ -185,6 +185,29 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
 * **For advanced users:**
   * You can remove all the tags of an employee with `edit INDEX t/` (see warning above)
 
+#### Removing an employee record : `delete`
+
+1. Get the [index](#glossary) of the employee under the [employee list](#glossary). View this image in [quick start](#quick-start) for more information.
+<box type="info" seamless>
+If the employee is not found, consider using <a href= "#listing-all-employees-list">list</a> or any <a href= "#finding-employees">find commands</a> to locate the employee in the employee list.
+</box>
+2. Type in the following [command](#glossary) in the [command box](#glossary) `delete INDEX` where `INDEX` is to be replaced with the index of the employee in the list currently
+ - For instance, to remove the whole record of the employee indexed 1, type `delete 1` to the command box.
+![Before Delete](images/before-delete.png)
```

Comment:
Note for future change - might be useful to either label the images or explain in the text what is going on, since there is quite a lot happening in the images